lib/nerves_hub_link/configurator.ex

```diff
@@ -118,11 +121,13 @@ defmodule NervesHubLink.Configurator do
   end
 
   defp fetch_default() do
-    if Code.ensure_loaded?(NervesKey) do
+    default = if Code.ensure_loaded?(NervesKey) do
       NervesHubLink.Configurator.NervesKey
     else
-      Default
+      NervesHubLink.Configurator.SharedSecret
     end
+    Application.put_env(:nerves_hub_link, :configurator, default)
```

Comment:
Why are we changing to put this back into the Application config?

Reply:
It's because of https://github.com/nerves-hub/nerves_hub_link/blob/main/lib/nerves_hub_link/socket.ex#L424-L425
And since there are two defaults, we need to know which was is being used.